packages/chain-adapters/src/cosmossdk/cosmos/CosmosChainAdapter.ts

```diff
@@ -109,13 +114,16 @@ export class ChainAdapter extends CosmosSdkBaseAdapter<KnownChainIds.CosmosMainn
       const {
         to,
         wallet,
-        bip44Params = this.defaultBIP44Params,
+        bip44Params,
         chainSpecific: { gas, fee },
         sendMax = false,
         value,
         memo = '',
       } = tx
 
+      if (!bip44Params) {
+        throw new Error('bip44Params are required in Cosmos SDK getAddress')
+      }
```

Comment:
This will go away in a follow-up PR, making `bip44Params` required will bork swappers